magicindicator/src/main/java/net/lucode/hackware/magicindicator/buildins/commonnavigator/titles/SimplePagerTitleView.java

```diff
@@ -54,7 +54,16 @@ public void onEnter(int index, int totalCount, float enterPercent, boolean leftT
     @Override
     public int getContentLeft() {
         Rect bound = new Rect();
-        getPaint().getTextBounds(getText().toString(), 0, getText().length(), bound);
+        String longestString = "";
+        if (getText().toString().contains("\n")) {
+            String[] brokenStrings = getText().toString().split("\\n");
+            for (String each : brokenStrings) {
+                if (each.length() > longestString.length()) longestString = each;
+            }
+        } else {
+            longestString = getText().toString();
+        }
+        getPaint().getTextBounds(longestString, 0, longestString.length(), bound);

```

Comment:
may be you should measure every broken string，the longest string which has max length not means max width after measured. Forgive my poor English.